src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -38,6 +42,36 @@ shared_ptr<Model> convert_model(const string& model_path) {
 
     return model;
 }
+
+ov::OutputVector fake_translator_ragged_tensor_to_sparse(const ov::frontend::NodeContext& node) {
+    // NOTE: pay attention that this is a fake translator for RaggedTensorToSparse
+    // only serves for testing purposes
+    FRONT_END_GENERAL_CHECK(node.get_input_size() > 1, "RaggedTensorToSparse expects at least two inputs.");
+    auto node_name = node.get_name();
+    auto row_splits = node.get_input(0);
+    auto strings = node.get_input(1);
+
+    // Override type of input tensor if this is a Parameter
+    if (auto parameter = as_type_ptr<ov::opset10::Parameter>(strings.get_node_shared_ptr())) {
+        parameter->set_partial_shape(ov::PartialShape{Dimension()});
+        parameter->set_element_type(ov::element::u8);
+        parameter->validate_and_infer_types();
+    }
+
+    row_splits = make_shared<ConvertLike>(row_splits, strings);
+    auto const_one = make_shared<Constant>(row_splits.get_element_type(), Shape{}, 1);
+    Output<Node> mul = make_shared<Multiply>(row_splits, const_one);
+    auto const_two = make_shared<Constant>(ov::element::u8, Shape{}, 2);
```

Comment:
row_splits.get_element_type() can be different from u8, in case of strings is Constant with other type.
maybe it's better to check this or use strings.get_element_type

Reply:
`const_two` is used for addition with strings for which we re-write its type to `u8` so they should be aligned.